ktxspec.adoc

```diff
@@ -1930,6 +1981,39 @@ seconds is stem:[duration / timescale].
 This metadata entry must not be used together with
 <<KTXcubemapIncomplete,`KTXcubemapIncomplete`>>.
 
+=== KTXmapRange
+When the payload is a floating point format with a restricted range,
+such as ASTC HDR, RGB9E5 and FLOAT16, and data is input from a
+source with 32-bit floating point data, such as a .exr file,
+the values in the source may be mapped to the more limited range.
+KTX file writers may indicate the mapping by using the key
+
+- `KTXmapRange`
+
+The value is 8 bytes representing two IEEE 754 single-precision
+finite floating-point values:
+[source,c]
+----
+Float32 scale
+Float32 offset
+----
```

Comment:
To make the spec language more normative, this part should be written like:
> The value is 8 bytes representing two IEEE 754 single-precision floating-point values:
> _(source block)_
> These values must be finite.

The use of "must" here has a normative meaning, i.e., if any of these values is not finite - the file is definitely invalid.

Reply:
Fixed in  https://github.com/KhronosGroup/KTX-Specification/pull/216/commits/fefab794e0728570fe994c05db2d60817962907b.